### PR TITLE
feat(translation): add description for source language

### PR DESCRIPTION
**Pull request by tamasdomokos**

### CODE REVIEW

**Comment by saghul** [comment]
Jenkins please test this please.

File: lang/main.json [hunk 1]
```diff
@@ -1225,6 +1225,8 @@
         "labelToolTip": "The meeting is being transcribed",
         "off": "Transcribing stopped",
         "pending": "Preparing to transcribe the meeting...",
+        "sourceLanguageDesc": "Currently the meeting language is set to <b>{{sourceLanguage}}</b>. <br/> You can change it from {{here}}",
```
Comment (horymury) [inline lang/main.json:1228]: Seems that `here` placeholder is not defined when  this translation is used.

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx [hunk 2]
```diff
@@ -65,11 +92,22 @@ const LanguageSelectorDialog = ({
...
             cancel = {{ hidden: true }}
             ok = {{ hidden: true }}
             titleKey = 'transcribing.subtitles'>
+            <p className = { styles.paragraphWrapper } >
+                { translateToHTML(t, 'transcribing.sourceLanguageDesc', {
+                    'sourceLanguage': t(`languages:${_sourceLanguage}`).toLowerCase()
+                }) }<span
+                    className = { styles.spanWrapper }
+                    onClick = { onClickSourceLanguage }>{t('transcribing.sourceLanguageHere')}.</span>
+            </p>
```
Comment (horymury) [inline react/features/subtitles/components/LanguageSelectorDialog.web.tsx:110]: doesn't something like this work?
```
            <p className = { styles.paragraphWrapper } >
                 {t('transcribing.sourceLanguageDesc', {
                    'sourceLanguage': t(`languages:${_sourceLanguage}`).toLowerCase()
                })}<span
                    className = { styles.spanWrapper }
                    onClick = { onClickSourceLanguage }>{t('transcribing.sourceLanguageHere')}.</span>
            </p>
```

Comment (horymury) [reply react/features/subtitles/components/LanguageSelectorDialog.web.tsx:110]: nevermind :)

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx [hunk 3]
```diff
@@ -93,13 +131,15 @@ function mapStateToProps(state: IState) {
...
         _conference: conference,
         _language,
+        _sourceLanguage: sourceLanguage,
         _translationLanguages: languages,
         _translationLanguagesHead: languagesHead
     };
 }
 
-export default connect(mapStateToProps)(LanguageSelectorDialog);
+export default connect(mapStateToProps)(translate(LanguageSelectorDialog));
```
[the remark this answers is not in the document]
Comment (tamasdomokos) [reply react/features/subtitles/components/LanguageSelectorDialog.web.tsx]: done

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx [hunk 4]
```diff
@@ -93,13 +131,15 @@ function mapStateToProps(state: IState) {
 
     const languages = transcription?.translationLanguages ?? TRANSLATION_LANGUAGES;
     const languagesHead = transcription?.translationLanguagesHead ?? TRANSLATION_LANGUAGES_HEAD;
+    const sourceLanguage = i18next.language;
```
[the remark this answers is not in the document]
Comment (tamasdomokos) [reply react/features/subtitles/components/LanguageSelectorDialog.web.tsx]: ok

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx [hunk 5]
```diff
@@ -65,11 +92,22 @@ const LanguageSelectorDialog = ({
         dispatch(toggleLanguageSelectorDialog());
     }, [ _language ]);
 
+    const onClickSourceLanguage = useCallback(() => {
```
[the remark this answers is not in the document]
Comment (tamasdomokos) [reply react/features/subtitles/components/LanguageSelectorDialog.web.tsx]: ok

File: lang/main.json (hunk 1 above)
Comment (tamasdomokos) [reply lang/main.json]: removed

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx [hunk 6]
```diff
@@ -1,35 +1,65 @@
...
+// @ts-ignore
 import { setRequestingSubtitles, toggleLanguageSelectorDialog, updateTranslationLanguage } from '../actions';
 
 import LanguageList from './LanguageList';
 
-interface ILanguageSelectorDialogProps {
+
+interface ILanguageSelectorDialogProps extends WithTranslation {
     _language: string;
+    _sourceLanguage: string;
```
Comment (horymury) [inline react/features/subtitles/components/LanguageSelectorDialog.web.tsx:27]: remove this

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx (hunk 6 above)
Comment (tamasdomokos) [reply react/features/subtitles/components/LanguageSelectorDialog.web.tsx]: ok

**Review by horymury — APPROVED** [review]

**Pull request merged**